### PR TITLE
Update Docs: Pastanaga UI and Quanta

### PR DESCRIPTION
**Pull request by jensens**
After a chat with @sneridagh about Quanta, I took the information and updated the outdated page .

### CODE REVIEW

**Comment by stevepiercy** [comment]
The news folder for Volto moved since it became a monorepo to https://github.com/plone/volto/tree/main/packages/volto/news. Docs are for all the packages in the repo, which is why `docs/` remains at the repo root.

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ jobs:
               - 'packages/scripts/**'
             voltoSlate:
               - 'packages/volto-slate/**'
+            documentation:
+              - 'packages/docs/**'
```
Comment (stevepiercy) [inline .github/workflows/changelog.yml:44]: Please revert any relocation of docs. It is for all packages in the repo, and it is not a separate package unto itself. Moving it will also break the main documentation.

File: .github/workflows/changelog.yml [hunk 2]
```diff
@@ -110,6 +112,14 @@ jobs:
         env:
           BASE_BRANCH: ${{ github.base_ref }}
 
+          - name: documentation changelog check
+        if: steps.filter.outputs.documentation == 'true'
+        run: |
+          git fetch --no-tags origin main
+          towncrier check --compare-with origin/main --dir docs
+        env:
+          BASE_BRANCH: ${{ github.base_ref }}
```
Comment (stevepiercy) [inline .github/workflows/changelog.yml:121]: Please revert.

File: docs/source/contributing/style-guide.md [hunk 3]
```diff
@@ -11,36 +11,47 @@ myst:
...
-As stated at the beginning, a design system is a continuous effort, that evolves over time with the users' feedback and needs.
+Volto, and thus Plone 6, underwent a gradual evolution of its design system over multiple years.
 
-Albert gifted the community with Pastanaga UI some years ago, which was implemented during last years as default theme UI and UX in Volto itself. Although unfinished, it provided us with endless resources and patterns that we applied to Volto bit by bit.
+## Volto and Pastanaga UI
 
-Lately, Albert has evolved Pastanaga to match the current trends, the feedback of these last years and the lessons learned developing Volto projects. So he has evolved Pastanaga and created Quanta. You can think in Quanta in a PastanagaUI 2.0.
-
-You can find some excerpts of how Quanta looks like in this PDF file: {download}`Quanta Spring 2021 <Quanta.pdf>`
+Around 2017, Albert Albert Casado Celma introduced Pastanaga UI, which became Volto's default theme UI/UX.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:22]:
```suggestion
Around 2017, Albert Casado Celma introduced Pastanaga UI, which became Volto's default theme UI/UX.
```

File: docs/source/contributing/style-guide.md [hunk 4]
```diff
@@ -11,36 +11,47 @@ myst:
...
+Despite being unfinished, it provided valuable resources and patterns gradually integrated into Volto.
+Albert has played a crucial role in refining Volto's UI/UX, offering valuable insights and guidance.
+As mentioned earlier, design systems evolve over time based on user feedback and changing needs.
 
 ## Tooling around a design system
 
-Volto adopted some time ago [Storybook](https://storybook.js.org), a tool that is specific for build and show off the visual elements that a design system has in isolation. Storybook provides a sandbox to build and test these visual elements (components) in isolation, mock them to show different data and test edge cases and at the same time document them, on how to instantiate them and use them for the developers' own benefit.
+Volto adopted some time ago [Storybook](https://storybook.js.org), a tool that is specific for build and show off the visual elements that a design system has in isolation.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:29]:
```suggestion
Volto adopted some time ago [Storybook](https://storybook.js.org), a tool that interactively displays the visual elements of a design system.
```

File: docs/source/contributing/style-guide.md [hunk 5]
```diff
@@ -11,36 +11,47 @@ myst:
...
+Storybook provides a sandbox to build and test these visual elements (components) in isolation, mock them to show different data and test edge cases and at the same time document them, on how to instantiate them and use them for the developers' own benefit.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:30]:
```suggestion
Storybook provides a sandbox to build and test these visual elements (components) in isolation, mock them to show different data, and test edge cases.
At the same time, Storybook documents components for how to instantiate them and use them in development.
```

File: docs/source/contributing/style-guide.md [hunk 6]
```diff
@@ -11,36 +11,47 @@ myst:
...
 
 Not all Volto's visual components are covered yet in Storybook, but it has already a good foundation.
+Look at the current components [Storybook build](https://6.docs.plone.org/storybook).
+
+## Quanta, the design system for Plone 7
 
-You can find the current Storybook build in: https://6.docs.plone.org/storybook
+Recently, Albert updated Pastanaga UI to align with current trends and feedback, creating Quanta — a version 2.0 of Pastanaga UI.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:37]:
```suggestion
Recently, Albert updated Pastanaga UI to align with current trends and feedback, creating Quanta, a version 2.0 of Pastanaga UI.
```

File: docs/source/contributing/style-guide.md [hunk 7]
```diff
@@ -11,36 +11,47 @@ myst:
...
+This reflects ongoing refinement and enhancement of the design system.
+Check out Quanta's appearance in this PDF: {download}`Quanta Spring 2021 <Quanta.pdf>`
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:39]:
```suggestion
Check out Quanta's appearance in this PDF: {download}`Quanta Spring 2021 <Quanta.pdf>`.
```

File: docs/source/contributing/style-guide.md [hunk 8]
```diff
@@ -11,36 +11,47 @@ myst:
...
+Although the timeline for Quanta's integration, potentially in Plone 7, remains uncertain, progress is being made on experimental packages.
+Generating interest within the community is also a priority.
 
-Quanta for now have the shape of a Volto addon: https://github.com/plone/volto-quanta
+In the meantime, for upcoming projects, Volto with Pastanaga UI remains the recommended and viable option.
 
 ## Help to make Quanta a reality
 
-Do you want to help Quanta a reality in Plone 6? Just ping the Volto Team in Github or in the Plone organization Volto Slack channel: https://plone.slack.com/?redir=%2Farchives%2FCQLBSUKRT to ask for access to the Quanta specs.
+Do you want to help Quanta a reality?
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:56]:
```suggestion
Do you want to help Quanta become a reality?
```

File: docs/source/contributing/style-guide.md [hunk 9]
```diff
@@ -11,36 +11,47 @@ myst:
...
+Just ping the Volto Team at GitHub or in the [Plone organization Discord](https://plone.org/community/chat) channel `#frontend-volto` to ask for access to the Quanta specs and how to help.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:57]:
```suggestion
Just ping the Volto Team on GitHub or in the [Plone organization Discord](https://plone.org/community/chat) channel [`#frontend-volto`](https://discord.com/channels/786421998426521600/787308038050545666) to ask for access to the Quanta specs and how to help.
```

**Comment by jensens** [comment]
> The news folder for Volto moved since it became a monorepo to https://github.com/plone/volto/tree/main/packages/volto/news. Docs are for all the packages in the repo, which is why `docs/` remains at the repo root.

This is why I added an allow-check for docs and looked how `prereleaser.js` works, which globs`'**/news/*.*(breaking|feature|bugfix|documentation|internal)'`, finding the docs/news then as well. This should be fine this way and then all news folders are in it's place.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
See comments about reverting check and where to put news items.

Otherwise, a few minor grammar and syntax suggestions.

Thank you for taking this on. It really needed some polishing from someone who knew the history.

**Comment by stevepiercy** [comment]
> This is why I added an allow-check for docs and looked how `prereleaser.js` works, which globs`'**/news/*.*(breaking|feature|bugfix|documentation|internal)'`, finding the docs/news then as well. This should be fine this way and then all news folders are in it's place.

We already put documentation change log entries in the news items directory in `packages/volto/news`. We don't need another check that will not be used. It's also out of scope for this pull request, and should be discussed amongst the Volto team before making changes of this nature.

File: .github/workflows/changelog.yml (hunk 1 above)
Comment (jensens) [reply .github/workflows/changelog.yml:44]: ah, this is wrong, i did not rlocate it, just forgot to remove the leading packagces.
```suggestion
              - 'docs/**'
```

File: .github/workflows/changelog.yml (hunk 2 above)
Comment (jensens) [reply .github/workflows/changelog.yml:121]: so, where to put then the changelog for docs if not under `/docs/news`?

Comment (stevepiercy) [reply .github/workflows/changelog.yml:121]: `packages/volto/news`

Comment (jensens) [reply .github/workflows/changelog.yml]: Done, moved and cleaned the mess up into one commit.

**Comment by jensens** [comment]
> We already put documentation change log entries in the news items directory in `packages/volto/news`

Since the placement of the news files are very different from other parts of Plone, I would like to see a explanation at https://6.docs.plone.org/volto/contributing/index.html#change-log-entry - its not obvious.

**Comment by stevepiercy** [comment]
> > We already put documentation change log entries in the news items directory in `packages/volto/news`
> 
> Since the placement of the news files are very different from other parts of Plone, I would like to see an explanation at https://6.docs.plone.org/volto/contributing/index.html#change-log-entry - its not obvious.

Screenshot 2024-03-21 at 2 44 15 PM

The "see also" link goes to https://6.docs.plone.org/contributing/index.html#change-log-entry which provides sufficient detail.

> For Volto, its repository is in a monorepo structure, consisting of several packages in the `packages` folder. Thus for Volto and its packages, change log entries should be created in `packages/PACKAGE_NAME/news/`, which is the root of the package.

I can add your suggestion, if you still think it would be beneficial. I would turn it into a snippet and include it, so that we don't use copy-paste. Please let me know. Thank you!

**Review by stevepiercy — APPROVED** [review]
This is professionally and well-written. Thank you!

**Comment by stevepiercy** [comment]
@sneridagh I'd like your review for accuracy, and let you take the merge honors.

**Comment by jensens** [comment]
> > > We already put documentation change log entries in the news items directory in `packages/volto/news`
> > 
> > 
> > Since the placement of the news files are very different from other parts of Plone, I would like to see an explanation at https://6.docs.plone.org/volto/contributing/index.html#change-log-entry - its not obvious.
> 
> Screenshot 2024-03-21 at 2 44 15 PM
> 
> The "see also" link goes to https://6.docs.plone.org/contributing/index.html#change-log-entry which provides sufficient detail.
> 
> > For Volto, its repository is in a monorepo structure, consisting of several packages in the `packages` folder. Thus for Volto and its packages, change log entries should be created in `packages/PACKAGE_NAME/news/`, which is the root of the package.
> 
> I can add your suggestion, if you still think it would be beneficial. I would turn it into a snippet and include it, so that we don't use copy-paste. Please let me know. Thank you!

Something like "in case of setup/CI/CD or documentation changes not bound to a specific package, use the PACKAGE_NAME=volto"?

**Review by sneridagh — APPROVED** [review]

**Comment by sneridagh** [comment]
> Something like "in case of setup/CI/CD or documentation changes not bound to a specific package, use the PACKAGE_NAME=volto"?

I'd be +1 on clarifying this.

@jensens very well written, it's just awesome. Thanks a lot!
